mlflow/azureml/cli.py

```diff
@@ -79,7 +79,7 @@ def commands():
         " that include the model path, the model run id (if specified), and more."
     ),
 )
-@experimental
+@deprecated(since="1.19.0")
```

Comment:
Can we add the `alternative` parameter here, as we did for the `deploy()` API?

Reply:
There is no alternative for this, we've completely deprecated it on our end.

Reply:
@trangevi `build_image()` was used as a step in the AzureML deployment procedure, so it makes sense to direct users towards the new deployment procedure, even if the new procedure doesn't involve a separate image building step. Can we specify the deployment plugin as an alternative?

Reply:
Done